src/templates/base/2019/contributors.html

```diff
@@ -154,6 +154,11 @@
 .contributor-team:last-child::after {
   content: '';
 }
+{% if language.lang_attribute in ['ja'] %}
```

Comment:
Is there any harm to doing this for every language?

And can we add to existing definition on line 146?

Reply:
We can actually. Given that we never do want it to wrap to the next line. And in that case definitely can be moved to line 146

Reply:
Done

Reply:
Something like: 
[lang="ja"] .contributor-team  {
  white-space: nowrap;
}
would be just enough in my opinion.

Reply:
It may not be needed for non-JA languages but I agree with the approach taken here to apply the styles evenly.